nix/gecko_env.nix

```diff
@@ -33,10 +33,10 @@ let
       owner = "eqrion";
       repo = "cbindgen";
       rev = "v${version}";
```

Comment:
@garbas do you know why the build didn't fail when we updated the version (which is used to retrieve the source code at a given revision) and forgot to update the hash?